samples/client/petstore/java/okhttp-gson/pom.xml

```diff
@@ -24,9 +24,9 @@
 
     <developers>
         <developer>
-            <name>OpenAPI-Generator Contributors</name>
```

Comment:
@burberius is there by any chance your PR is based on a commit a while ago? Some of these changes in pom.xml (e.g. removal of src/main/java) do not look right to me.

If that's the case, I would suggest filing a new PR based on the latest master instead.